### DISCUSSION TITLE
Anyone know how to disable popup password for google?

### DISCUSSION THREAD

**Discussion by cuppjj** [ultrafunkamsterdam/undetected-chromedriver#1264] [General]
driver = uc.Chrome(enable_cdp_events=True, headless=True, version_main=113, disablegpu=True, no_sandbox=True)
driver.maximize_window()
driver.get("url)

Im using this code and cant seem to find a way to disable the remember ur password pop up? Anyone have a code that is working with this library that wouldnt mind sending it over?

**Comment by montovaneli** [comment]
```
options = uc.ChromeOptions()
# option for not showing save password prompt
options.add_argument("--password-store=basic")
options.add_experimental_option(
    "prefs",
    {
        "credentials_enable_service": False,
        "profile.password_manager_enabled": False,
    },
)

options.add_argument('--no-sandbox')
options.add_argument('--disable-gpu')
```

I also suggest to use the new headless mode instead of headless=True argument
`options.add_argument('--headless=new')`

Finally:

`driver = uc.Chrome(options=options, enable_cdp_events=True, version_main=113)`
